src/main/java/com/questhelper/helpers/quests/legendsquest/LegendsQuest.java

```diff
@@ -561,6 +579,9 @@ public void setupRequirements()
 
 		chargeOrbRunes = new ItemRequirements(LogicType.AND, "Runes for any charge orb spell you have the level to cast", cosmic3, elemental30);
 
+		teleToLegendsGuildHint = new ItemRequirement("Fairy ring BLR, or Ardougne Teleport then run north-east", -1, -1);
```

Comment:
WDYT of some contextual hints for the bank like so? We'd also need to import VarbitRequirement, QuestVarbits, and Operation.

```suggestion
		ItemRequirement teleToLegendsGuildFairyRing = new ItemRequirement("Fairy ring to Legends' Guild (BLR)", ItemCollections.FAIRY_STAFF)
			.showConditioned(new VarbitRequirement(QuestVarbits.QUEST_FAIRYTALE_II_CURE_A_QUEEN.getId(),
			Operation.GREATER_EQUAL, 40, "Partial completion of Fairytale II for access to fairy rings"));
		ItemRequirement teleToLegendsGuildArdougne = new ItemRequirement("Ardougne teleport and run north-west to Legends' Guild", ItemID.ARDOUGNE_TELEPORT);
		teleToLegendsGuildHint = new ItemRequirements(LogicType.OR,
			"Fairy ring BLR, or Ardougne Teleport then run north-east",
			teleToLegendsGuildFairyRing,
			teleToLegendsGuildArdougne);
```

Reply:
Tested this with the PanelDetails change in [`c33cf7a` (#1275)](https://github.com/Zoinkwiz/quest-helper/pull/1275/commits/c33cf7a688e4c47cca4609261ae6723f22900a1a) - works for me!